app/models/preneeds/military_rank.rb

```diff
@@ -3,10 +3,28 @@
 require 'common/models/base'
 
 module Preneeds
+  # Models a military rank from the EOAS service.
+  # For use within the {Preneeds::BurialForm} form.
+  #
+  # @!attribute branch_of_service_cd
+  #   @return [String] brach of service abbreviated code
+  # @!attribute military_rank_detail
+  #   @return [MilitaryRankDetail] rank details object
+  # @!attribute activated_one_date
+  #   @return [Common::UTCTime] activated date one
+  # @!attribute activated_two_date
+  #   @return [Common::UTCTime] activated date two
+  # @!attribute activated_three_date
+  #   @return [Common::UTCTime] activated date three
+  # @!attribute deactivated_one_date
+  #   @return [Common::UTCTime] deactivated date one
+  # @!attribute deactivated_two_date
+  #   @return [Common::UTCTime] deactivated date two
+  # @!attribute deactivated_three_date
+  #   @return [Common::UTCTime] deactivated date three
+  #
   class MilitaryRank < Common::Base
     attribute :branch_of_service_cd, String
-    attribute :officer_ind, String
```

Comment:
Could not find this attribute in any examples or code.
[Example doc.](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Pre-Needs-Burial/api/samples/getMilitaryRankForBranchOfService.md)